### PR TITLE
Fixes #934: Allow custom headers to pass through CORS

### LINKED ISSUE #934: Allow custom headers to pass through CORS
**What would you like to be added**:
We're unable to fetch custom headers right now for some reason. After lots of work by @manekenpix we found out it was a CORS related setting.

**Why would you like this to be added**:
We have custom headers but we're unable to get them and can only get 'Content-Type' at the moment.

### PR DESCRIPTION
**Pull request by manekenpix**
Co-authored: Calvin <calvinho125@gmail.com>

## Issue This PR Addresses

Fixes #934 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [x] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

This PR exposes `X-Total-Count` using cors options. It's required for #931 

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by c3ho** [comment]
Relevant article to `app.options` was removed in `app.js`
https://stackoverflow.com/questions/52942286/comparing-implementing-app-usecorscorsoptions-and-app-options-cors

File: src/backend/web/app.js [hunk 1]
```diff
@@ -22,8 +22,10 @@ const app = express();
 app.use(helmet());
 
 // Enable CORS and preflight checks on all routes
-app.use(cors());
-app.options('*', cors());
```
Comment (humphd) [inline src/backend/web/app.js:26]: Why remove this?  For preflight checks that need to request options, we'd need this.

Comment (c3ho) [reply src/backend/web/app.js:26]: @humphd 
image

Doesn't this not mean they are doing the same thing?

Comment (humphd) [reply src/backend/web/app.js:26]: You're right, thanks for helping me understand.

**Pull request merged**